models/eos/ospfv3/eos_ospfv3.yaml

```diff
@@ -0,0 +1,405 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: eos
```

Comment:
The ANSIBLE_METADATA doesn't have to be here anymore. I don't know if the updated RMB tool still includes it or not.